travel.py

```diff
@@ -52,7 +52,7 @@
 app.config.from_object(__name__)
 app.config['CORS_HEADERS'] = 'Content-Type'
 app.config['SWAGGER'] = {
-    'openapi': '3.0.5',
+    'openapi': '3.0.3',
```

Comment:
Dunno where I got `3.0.5` from 😆  - shame on this library for not telling me about it though.